pages/docs/middleware.en-US.md

```diff
@@ -28,6 +30,24 @@ function myMiddleware (useSWRNext) {
 }
 ```
 
+Typescipt
+
+```jsx
+import { Middleware } from "swr/dist/types";
+
+export const myMiddleware: Middleware = (useSWRNext) => {
```

Comment:
We have a [typescript page](https://swr.vercel.app/docs/typescript#middleware-types), maybe you could leave a link to it instead of duplicating it here?

Reply:
Done